packages/theme-chalk/src/splitter.scss

```
@@ -69,7 +70,7 @@
     }
 
     &:hover {
-      &::before {
+      &::before:not(.is-disabled) {
```

The `is-disabled` class is applied to the `splitter-bar`.

```suggestion
      &:not(.is-disabled)::before {
```